File: ios/RCTMGL-v10/RCTMGLRasterLayerManager.m
#import <React/RCTBridgeModule.h>
#import <React/RCTViewManager.h>

@interface RCT_EXTERN_MODULE(RCTMGLRasterLayerManager, RCTViewManager)

// symbol layer props
RCT_EXPORT_VIEW_PROPERTY(sourceLayerID, NSString)

// standard layer props
#include "CommonLayerProperties.H"

@end

File: ios/RCTMGL-v10/RCTMGLRasterLayerManager.swift
@objc(RCTMGLRasterLayerManager)
class RCTMGLRasterLayerManager: RCTViewManager {
    @objc
    override static func requiresMainQueueSetup() -> Bool {
        return true
    }
  
    override func view() -> UIView! {
      let layer = RCTMGLRasterLayer()
      layer.bridge = self.bridge
      return layer
    }
}
